_includes/payment-order-other-features.md

```diff
@@ -279,16 +283,36 @@ for future purchases. The payer is, by default, asked if they want to store
 their payment details, so even with `generatePaymentToken` set to `true`, it is
 still up to the payer if they want the details stored or not.
 
-{% include alert.html type="informative" icon="info" body="Please note that not
-all payment instruments provided by Swedbank Pay support Payer Awareness today."
+{% include alert.html type="informative" icon="info" body="Remember that the
+responsibility to enforce the GDPR requirements lies on you as a merchant."
 %}
```

Comment:
Don't replace the content of this alert. We should have two alerts for this section:

1. Explaining that Payer Awareness is not available in all payment instruments.
2. Detailing the GDPR responsibility.

These two alerts need to be placed apart so they aren't displayed right after each other, as that would look weird. So that's why I suggested to move the existing alert up and create a new alert for the new GDPR section.

Reply:
Ah, that was weird. I now see the first alert as well. The placement is perfect, please just ignore me. 😅